graphql/util.go

```diff
@@ -1224,3 +1224,60 @@ func collapseCommit(ctx context.Context, mainlineCommits MainlineCommits, mainli
 		mainlineCommitVersion.RolledUpVersions = []*restModel.APIVersion{&apiVersion}
 	}
 }
+
+func getProjectPermissionLevel(projectPermission ProjectPermission, access AccessLevel) (requiredPermission string, requiredLevel int, err error) {
```

Comment:
although this is a private function and doesn't require a function header, i think it could still be helpful to have a comment that denotes that this function just translates the graphql enums to app settings (this isn't blocking, disregard if you don't think so!)